themes/default/layouts/webinars/single.html

```diff
@@ -67,6 +67,8 @@ <h4 class="text-orange">Join us to learn:</h4>
                                 {{ end }}
                             </ul>
                         {{ end }}
+                        <h4>Prerequisites</h4>
+                        <div>This course will be taught using Pulumi Cloud. To follow along, sign up for your free account: <a class="text-blue-700 font-semibold underline hover:cursor-pointer" href="https://app.pulumi.com/signup">https://app.pulumi.com/signup</a></div>
```

Comment:
should we just have the make the text "sign up" a link itself, instead of having the url in the page?

Reply:
@lindsaymarolich  and I talked about that and concluded we should be on the more transparent side with this one